src/index.es6

```diff
@@ -130,6 +130,15 @@ module.exports.test = function(options) {
         tempDir: options.reportDir
       });
       runTestsPromise.done(result => {
+        serverProcess.on('close', () => {
+            setTimeout(function(){
```

Comment:
callback() called inside a timeout because I detected that when it's called with an error it interrupts de generation of HTML gemini report. It's working this way, but I'm sure the code is improveable and reviews/suggestions accepted.